src/account-info/AccountDetailsScreen.js

```diff
@@ -16,17 +16,21 @@ class AccountDetailsScreen extends Component {
     avatarUrl: string,
   }
 
+  componentWillMount() {
+    this.email = this.props.email;
```

Comment:
Why move `email` to a member variable (instead of directly using `this.props.email` as before)?